### DISCUSSION TITLE
how to use the galaxy gate

### DISCUSSION THREAD

**Discussion by aixi-aizi** [Slimefun-Addon-Community/Galactifun#114] [General]
I'm glad to see such a creative and playable work, but I always have a question about how to use the galaxy gate and the galaxy gate controller in the galaxy system. I'd like to ask the author to answer it. Thank you .maybe one picture would hlep me most.

**Comment by Seggan** [comment]
![image](https://user-images.githubusercontent.com/49963543/152385496-c56ff6d1-66ed-4879-a24c-4f186b7d47a5.png)

Here is an image. The Stargate must be aligned on the Z axis

**Reply by aixi-aizi** [reply]
very thank you
